bin/kafka-server-start.sh

```diff
@@ -31,14 +31,28 @@ fi
 
 EXTRA_ARGS=${EXTRA_ARGS-'-name kafkaServer -loggc'}
 
+CONFDIR="${base_dir}/../config"
```

Comment:
I don't understand the need to pass in a CONFDIR here, Kafka should get all the config it needs from the properties file.